runtime/onert/core/src/compiler/StaticShapeInference.cc

```diff
@@ -182,6 +182,34 @@ void StaticShapeInferer::visit(const ir::operation::BatchMatMul &op)
   output.info().shape(new_shape);
 }
 
+void StaticShapeInferer::visit(const ir::operation::BCQFullyConnected &op)
+{
+  const auto input_idx{op.getInputs().at(ir::operation::BCQFullyConnected::Input::INPUT)};
+  const auto &input = _operands.at(input_idx);
+
+  const auto cluster_idx{
+      op.getInputs().at(ir::operation::BCQFullyConnected::Input::WEIGHTS_CLUSTERS)};
+  const auto &cluster = _operands.at(cluster_idx);
+
+  const auto output_idx = op.getOutputs().at(0);
+  ir::Operand &output = _operands.at(output_idx);
+
+  if (!cluster.isConstant())
```

Comment:
AFAIK, cluster is always constant.
Is this condition correct...?

Reply:
I have changed.